quantumdb-core/src/main/java/io/quantumdb/core/versioning/Changelog.java

```diff
@@ -23,7 +23,7 @@
  * This class allows you to define a series of successive ChangeSets, building up a changelog in the process.
  */
 @Data
-@EqualsAndHashCode(exclude = { "idGenerator" })
```

Comment:
Maybe avoid reformatting classes that are not really changed, as this pollutes the git annotations and git history a bit. I'm not against these changes, but they need to be enforced for everyone contributing to the codebase, otherwise people will just keep reformatting to whatever style they like.

Reply:
I only reformatted the files I did change but this should have been in the other pull request, my bad.